packages/web-app-files/src/components/PublicLinksSidebar/PrivateLinkItem.vue

```diff
@@ -39,7 +37,7 @@ export default {
   }),
 
   computed: {
-    ...mapGetters('Files', ['highlightedFile']),
+    ...mapGetters('Files', ['highlightedFile'], ['capabilities']),
```

Comment:
`mapGetters` expects two params: the namespace (can be omitted for root level) and **one** array or object. Please merge the two getter names into one array. ;-)